src/pages/IdeaDetailPage/ideaDetailPage.helper.ts

```diff
@@ -0,0 +1,11 @@
+import { IdeaModel } from 'pouchDB/api/idea';
+import { IdeaDoc } from 'pouchDB/types';
+// import { binarySearchOfPouchDbDocs } from 'utils/binarySearch';
```

Comment:
If its not unused can we remove this or you can give a justification about the usage plan

Reply:
```suggestion
```